web/src/pages/summary.rs

```diff
@@ -20,57 +20,59 @@ impl Component for Summary {
 
     fn view(&self) -> Html {
         html! { <>
-            <h1> { "Summary" } </h1>
+            <h1> { "Further informations" } </h1>
 
             <p> { "
                 We showed that AT2 is behaving as any other blockchain.
-                You can send asset with it, see other users on the network
-                and see the latest processed transactions.
-                But using a speedtest really uncovers the true power of AT2:
-                it is faster than other most popular blockchains, beating " }
-                <a href="https://bitcoin.org"> { "Bitcoin" } </a>
+                It can send asset, show other accounts on the network
+                and watch the latest processed transactions.
+                But using a speedtest reveals the true power of AT2:
+                it is faster than other two most popular blockchains, beating " }
+                <a href="https://bitcoin.org" target="_blank"> { "Bitcoin" } </a>
                 { " and " }
-                <a href="https://ethereum.org"> { "Ethereum" } </a>
+                <a href="https://ethereum.org" target="_blank"> { "Ethereum" } </a>
                 { " by a factor of 10.
```

Comment:
That's really not a lot. I thought that we saw a factor of 100, no? And theoretical performance is even more?

Also, latest developments in blockchains have more than 600kTPS: https://arxiv.org/abs/2105.11827

Reply:
Not changing it then.